# Simple YAML will be put in ROS parameters by launch file
# ref : https://yaml.org
fib_init:
- 0
- 2
fibonacci_max_number: 1024